src/test/java/runners/SignUpTest.java

```diff
@@ -0,0 +1,13 @@
+package runners;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(features = "src/test/resources/features/signup.feature"
+        , tags = "@EmptyFields"
```

Comment:
Is this tag necessary here? You are declaring some more tags inside the `.feature` file.

Reply:
I will remove this line.